ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController.swift

```diff
@@ -2392,7 +2392,7 @@ extension BrowserViewController: TabsBarViewControllerDelegate {
 }
 
 extension BrowserViewController: TabDelegate {
-  func tab(_ tab: Tab, didCreateWebView webView: WKWebView) {
+  func tab(_ tab: Tab, didCreateWebView webView: UIView) {
```

Comment:
Why UIView? Is there any other view that can be in a tab other than a WebView?

Reply:
There is no reason for this to be passing a WKWebView anymore since the receiver should never be assigning any of its delegates. In the future the tab will manage its own `UIView` container view that will house the web view and the BVC will add _that_ view to the hierarchy instead of the `WKWebView`